### PR TITLE
Mineral Rework

### PR DESCRIPTION
**Pull request by Dorodomki**
## About The Pull Request

reduces the ammount of gold that spawns on the minerals of high reward making them proper rare materials to find

## Why It's Good For The Game

the days of powergamers rushing MT decap or "troll cave" are ruined and the economy saved, won't be easy find +10 gold ores  like is right now

### CODE REVIEW

**Comment by Dorodomki** [comment]
the iron ores will continue as they are since you can get plenty already, flood the market with iron isn't good either

**Comment by ThePainkiller** [comment]
We already have silver as rare as this PR makes gold, at the very much reduce it to like 10 but don't jump into insta rarity territory

File: code/game/turfs/closed/minerals.dm [hunk 1]
```diff
@@ -602,19 +602,19 @@
...
-	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/salt = 15)
+	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/salt = 15, /turf/closed/mineral/rogue/copper = 10, /turf/closed/mineral/rogue/iron = 5)
 	mineralChance = 30
 	max_integrity = 400
 
 /turf/closed/mineral/random/rogue/med
 	icon_state = "minrandmed"
 	mineralChance = 50
-	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/salt = 25, /turf/closed/mineral/rogue/iron = 25, /turf/closed/mineral/rogue/coal = 25)
+	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/salt = 25, /turf/closed/mineral/rogue/iron = 25, /turf/closed/mineral/rogue/coal = 25, /turf/closed/mineral/rogue/copper = 25)
```
Comment (ThePainkiller) [inline code/game/turfs/closed/minerals.dm:612]:
```suggestion
	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/salt = 20, /turf/closed/mineral/rogue/iron = 25, /turf/closed/mineral/rogue/coal = 20, /turf/closed/mineral/rogue/copper = 10)
```

File: code/game/turfs/closed/minerals.dm [hunk 2]
```diff
@@ -602,19 +602,19 @@
...
 
 /turf/closed/mineral/random/rogue/high
 	icon_state = "minrandhigh"
 	mineralChance = 50
-	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/gold = 15 , /turf/closed/mineral/rogue/iron = 25,)
+	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/gold = 10 , /turf/closed/mineral/rogue/iron = 25, /turf/closed/mineral/rogue/silver = 5)
```
Comment (ThePainkiller) [inline code/game/turfs/closed/minerals.dm:617]:
```suggestion
	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/gold = 10 , /turf/closed/mineral/rogue/iron = 25, /turf/closed/mineral/rogue/silver = 10)
```

File: code/game/turfs/closed/minerals.dm [hunk 3]
```diff
@@ -639,13 +639,30 @@
 	spreadChance = 0
 	spread = 0
 
+
+/turf/closed/mineral/rogue/copper
+	desc = "seems rich in copper"
+	icon_state = "mingold"
+	mineralType = /obj/item/rogueore/copper
+	rockType = /obj/item/natural/rock/copper
+	spreadChance = 6
```
Comment (ThePainkiller) [inline code/game/turfs/closed/minerals.dm:648]:
```suggestion
	spreadChance = 4
```
We don't want them taking over a lot of other minerals like an invasive weed but it should still be common

File: code/game/turfs/closed/minerals.dm [hunk 4]
```diff
@@ -602,19 +602,19 @@
 	turf_type = /turf/open/floor/rogue/naturalstone
 	above_floor = /turf/open/floor/rogue/naturalstone
 	baseturfs = list(/turf/open/floor/rogue/naturalstone)
-	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/salt = 15)
+	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/salt = 15, /turf/closed/mineral/rogue/copper = 10, /turf/closed/mineral/rogue/iron = 5)
```
Comment (ThePainkiller) [inline code/game/turfs/closed/minerals.dm:605]:
```suggestion
	mineralSpawnChanceList = list(/turf/closed/mineral/rogue/salt = 20, /turf/closed/mineral/rogue/copper = 15, /turf/closed/mineral/rogue/iron = 5)
```

**Review by ThePainkiller — DISMISSED** [review]

**Comment by ThePainkiller** [comment]
Oh yes looking good now

**Review by ThePainkiller — APPROVED** [review]

**Pull request merged**